spec/api-ipc-renderer-spec.js

```diff
@@ -16,97 +20,97 @@ describe('ipc renderer module', () => {
   afterEach(() => closeWindow(w).then(() => { w = null }))
 
   describe('ipc.sender.send', () => {
-    it('should work when sending an object containing id property', (done) => {
+    it('should work when sending an object containing id property', done => {
       const obj = {
         id: 1,
         name: 'ly'
       }
-      ipcRenderer.once('message', function (event, message) {
-        assert.deepEqual(message, obj)
+      ipcRenderer.once('message', (event, message) => {
+        expect(message).to.deep.equal(obj)
         done()
       })
       ipcRenderer.send('message', obj)
     })
 
-    it('can send instances of Date', (done) => {
+    it('can send instances of Date', done => {
       const currentDate = new Date()
-      ipcRenderer.once('message', function (event, value) {
-        assert.equal(value, currentDate.toISOString())
+      ipcRenderer.once('message', (event, value) => {
+        expect(value).to.equal(currentDate.toISOString())
         done()
       })
       ipcRenderer.send('message', currentDate)
     })
 
-    it('can send instances of Buffer', (done) => {
+    it('can send instances of Buffer', done => {
       const buffer = Buffer.from('hello')
-      ipcRenderer.once('message', function (event, message) {
-        assert.ok(buffer.equals(message))
+      ipcRenderer.once('message', (event, message) => {
+        expect(buffer).to.equal(message)
```

Comment:
We have to use `expect(buffer.equals(message)).to.be.true()` here, Buffer is a complex object, so we have to compare them "by value".